react-search/my-app/src/SearchBar.tsx

```diff
@@ -0,0 +1,16 @@
+import { ChangeEvent } from 'react';
+
+type Props = {
+  handleSearchChange: (e: ChangeEvent<HTMLInputElement>) => void;
```

Comment:
Name should start with `on`.

It would be best to pass just the text instead of the event.